client/src/components/TextField/TextField.js

```diff
@@ -31,6 +31,7 @@ class TextField extends SilverStripeComponent {
       value: this.props.value,
       placeholder: this.props.placeholder,
       autoFocus: this.props.autoFocus,
+      required: Boolean(this.props.meta && this.props.meta.error),
```

Comment:
nice, do we need to repeat this for other fields as well?

Reply:
Yes for the fields that are not subclass of `TextField`. Depends on the other comment, I might need to replicate this in other field components.